internal/store/fleet.go

```diff
@@ -217,7 +226,37 @@ func (s *FleetStore) CreateOrUpdate(ctx context.Context, orgId uuid.UUID, resour
 	return &updatedResource, created, nil
 }
 
+func (s *FleetStore) CreateOrUpdateMultiple(ctx context.Context, orgId uuid.UUID, resources ...*api.Fleet) error {
```

Comment:
If the implementation is going to return on the first error could we just have a single method on the interface like this?

```
CreateOrUpdate(ctx context.Context, orgId uuid.UUID, fleets ...*api.Fleet) error